processes/ingest/chicago_isac.py

```diff
@@ -44,9 +51,9 @@ def processChicagoISACRecord(self, record):
             self.storePDFManifest(chicagoISACRec.record)
             self.addDCDWToUpdateList(chicagoISACRec)
             
-        except (MappingError, HTTPError, ConnectionError, IndexError, TypeError) as e:
+        except Exception as e:
             logger.exception(e)
```

Comment:
The nice thing with logger.exception is that it will log all of the error details for us, so I suggest adding a descriptive message here like unable to process ISAC record.

Reply:
Ok I can add a more descriptive message.